vignettes/ab-testing.Rmd

```diff
@@ -0,0 +1,479 @@
+---
+title: "Probabilistic A/B Testing with rstanarm"
+author: "Imad Ali"
+date: "`r Sys.Date()`"
+output:
+  html_vignette:
+    toc: yes
+params:
+  EVAL: !r identical(Sys.getenv("NOT_CRAN"), "true")
+bibliography: mrp-files/mrp.bib
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_chunk$set(collapse = TRUE)
+library(rstan)
+library(rstanarm)
+library(dplyr)
```

Comment:
Is it possible to do this without dplyr? I don't think we currently have dplyr as a dependency but we would need to add it to use it in the vignette.

Reply:
So I use dplyr [here](https://github.com/stan-dev/rstanarm/blob/doc/ab-testing/vignettes/ab-testing.Rmd#L408-L409). I don't have to use it and it's straightforward to rewrite this in non-dplyr code. But since it's pretty much a standard for industry workflow maybe it's something to consider using in all the vignettes. But then, like you mentioned, it's not a dependency for the main purpose of the package. Let me know what you want me to do.